### PR TITLE
Workflow update to node 20

### PR DESCRIPTION
**Pull request by KenAJoh**
### Description

Updated actions to use the more recent versions, where pretty much each of them made the major-change because of node v16 -> v20 update.

### CODE REVIEW

**Comment by HalvorHaugan** [comment]
Is this something new that comes with the new versions?
image

File: .github/workflows/aksel-redeploy.yml
Comment (HalvorHaugan) [inline .github/workflows/aksel-redeploy.yml:1]: Off topic, but the name should be corrected: "Redeploy image **to for** aksel.nav.no"

**Review by HalvorHaugan — DISMISSED** [review]
LGTM 🤷‍♂️

**Comment by KenAJoh** [comment]
> Is this something new that comes with the new versions? image

I enabled Code scanning earlier today here: https://github.com/navikt/aksel/settings/security_analysis, so unrelated to this PR

**Pull request merged**